lib/Channel.js

```diff
@@ -12,22 +17,99 @@ var Channel = Model.extend(volumeParams, {
   },
 
   collections: {
-    notes: Collection.extend({
-      model: Note
-    })
+    rawNotes: NotesCollection,
+    expandedNotes: NotesCollection
   },
 
   initialize: function () {
     Model.prototype.initialize.apply(this, arguments);
-    this.notes.on('start', this._start.bind(this));
-    this.notes.on('stop', this._stop.bind(this));
+
+    this._cache = {};
+
+    this.listenTo(this.rawNotes, 'start', this._start);
```

Comment:
as expanded note, as well as raw note, would be part of a collection that is a child of the channel, there's no need to do this even forwarding anymore. the only purpose of ghosts would be to have a representation of the expanded position and transformed values, with a one-way dataflow.